mock/src/mock/chain.rs

```diff
@@ -660,17 +660,15 @@ pub fn mock_chain_data(consumed_notes: Vec<Note>) -> (ChainMmr, Vec<InputNote>)
 // ================================================================================================
 
 /// Converts the MMR into partial MMR by copying all leaves from MMR to partial MMR.
-fn mmr_to_chain_mmr(mmr: &Mmr) -> ChainMmr {
+fn mmr_to_chain_mmr(mmr: &Mmr, blocks: &[BlockHeader]) -> ChainMmr {
     let num_leaves = mmr.forest();
     let mut partial_mmr = PartialMmr::from_peaks(mmr.peaks(mmr.forest()).unwrap());
```

Comment:
Mocks are only used in testing right? If not then it would seem that the number of `.unwrap()` in this function could be concerning.

Reply:
Yep - this is only for testing - so, `unwrap()'s are OK here.